ramls/items-lost-requiring-actual-costs.json

```diff
@@ -0,0 +1,202 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Collection of items",
+  "description": "Collection of items",
+  "type": "object",
+  "properties": {
+    "items": {
+      "description": "Paged collection of items",
+      "id": "items",
+      "type": "array",
+      "items": {
```

Comment:
Are all of these properties needed for the report?

Reply:
I am thinking I can shorten this to just the list of columns like those described in @bohdan-suprun 's comment above with the filtered list of loans.